Registry/registry.chem

```diff
@@ -20,6 +20,13 @@ state    real    QC_B4MP         ikj    misc        1         -      -        "Q
 state    real    QI_B4MP         ikj    misc        1         -      -        "QI_B4MP"             "CLOUD ICE BEFORE MICROPHYSICS"    "kg kg-1"
 state    real    QS_B4MP         ikj    misc        1         -      -        "QS_B4MP"             "SNOW BEFORE MICROPHYSICS"         "kg kg-1"
 
+package  do_mozart_ph_diag    mozart_ph_diag==1         -           state:ph_cw,ph_aer01,ph_aer02,ph_aer03,ph_aer04
```

Comment:
The package definitions start around line number 3888. For convention within this file, this package line should be moved.

Reply:
done